src/triage/component/catwalk/model_testers.py

```diff
@@ -41,6 +41,9 @@ def __init__(
             training_metric_groups=evaluator_config.get("training_metric_groups", []),
         )
 
+        self.subsets = evaluator_config.get("subsets")
```

Comment:
Yeah probably just cancel all the changes to this file and we'll remove the file in another PR, or just remove the file here.